ocaml/src/cli_asd.ml

```diff
@@ -582,11 +588,14 @@ let asd_statistics hosts port_o transport asd_id
                 | Some (claim_info, osd) ->
                   let conn_info =
                     let open Nsm_model.OsdInfo in
-                    let conn_info' = get_conn_info osd.kind in
+                    let conn_info' = match osd.kind with
+                      | Asd (x, _) -> x
+                      | Kinetic _ | Alba _ -> assert false
```

Comment:
This specific extraction happens exactly twice. I did it and it saved me exactly 0 lines of code.
I must say that it feels a bit like nitpicking if for the rest we pretend to not even care about the duplication damage inflicted upon the c++ proxy client in the rdma pull request.
(I may undo that damage someday though, it still bothers me.)